### PR TITLE
initial schema

### PR DESCRIPTION
**Pull request by Guseyn**

### CODE REVIEW

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this pull request

**Comment by Guseyn** [comment]
resolve #6

File: resources/liquibase/001-initial-schema.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      CREATE TABLE badge (
+        id SERIAL PRIMARY KEY,
+        project INT NOT NULL REFERENCES project(id),
+        text VARCHAR(64) NOT NULL,
+        created TIMESTAMPTZ DEFAULT now() NOT NULL
+      );
+      CREATE INDEX idx_badge1 ON badge (project);
+    </sql>
+    <sql>
+      CREATE TABLE badge_project (
```
Comment (yegor256) [inline resources/liquibase/001-initial-schema.xml]: @Guseyn what this is for?

**Comment by Guseyn** [comment]
@yegor256 done

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#46](https://github.com/yegor256/codexia/issues/46); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/codexia#46` is not assigned, can't get performer

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Pull request merged**